app/services/idv/analytics_events_enhancer.rb

```diff
@@ -1,61 +1,61 @@
 module Idv
   module AnalyticsEventsEnhancer
-    DECORATED_METHODS = %i[
-      idv_cancellation_confirmed
-      idv_cancellation_go_back
-      idv_cancellation_visited
-      idv_forgot_password
-      idv_forgot_password_confirmed
-      idv_final
-      idv_gpo_address_letter_enqueued
-      idv_gpo_address_letter_requested
-      idv_in_person_ready_to_verify_visit
-      idv_letter_enqueued_visit
-      idv_personal_key_acknowledgment_toggled
-      idv_personal_key_downloaded
-      idv_personal_key_submitted
-      idv_personal_key_visited
-      idv_phone_confirmation_form_submitted
-      idv_phone_confirmation_otp_rate_limit_attempts
-      idv_phone_confirmation_otp_rate_limit_locked_out
-      idv_phone_confirmation_otp_rate_limit_sends
-      idv_phone_confirmation_otp_resent
-      idv_phone_confirmation_otp_sent
-      idv_phone_confirmation_otp_submitted
-      idv_phone_confirmation_otp_visit
-      idv_phone_confirmation_vendor_submitted
-      idv_phone_error_visited
-      idv_phone_of_record_visited
-      idv_phone_otp_delivery_selection_visit
-      idv_phone_otp_delivery_selection_submitted
-      idv_proofing_resolution_result_missing
-      idv_enter_password_submitted
-      idv_enter_password_visited
-      idv_please_call_visited
-      idv_start_over
-    ].freeze
+    IGNORED_METHODS = %i[
```

Comment:
Right now this list is just any frontend events that were not previously "opted in". Hopefully big changes to this list can come in as separate PRs.

It could be argued that many events before document_capture don't benefit from having proofing_components added automatically. However:

* We have much better back button support than we did when this events enhancer was first written, so proofing_components could have data
* I'm planning on using this events enhancer to add more properties in the future that would be relevant for those earlier events